go-get.go

```diff
@@ -0,0 +1,6 @@
+// This file is here so go get succeeds, as without it errors with:
+// no buildable Go source files in ...
+//
+// +build !windows
+
+package wmi
```

Comment:
When I tested that was not the case the go get aborted at this error.


On 27 Jan 2015, at 08:19, Matt Jibson <notifications@github.com> wrote:

In go-get.go
<https://github.com/StackExchange/wmi/pull/7#discussion_r23592564>:

> @@ -0,0 +1,6 @@
> +// This file is here so go get succeeds, as without it errors with:
> +// no buildable Go source files in ...
> +//
> +// +build !windows
> +
> +package wmi

I'm not sure why this is necessary. I'm ok with those messages, since they
aren't errors, just warnings, and go get continues.

—
Reply to this email directly or view it on GitHub
<https://github.com/StackExchange/wmi/pull/7/files#r23592564>.